setup.py

```diff
@@ -25,4 +25,5 @@
         "syft==0.2.3a1",
         "requests==2.22.0",
     ],
+    extras_require={"test": ["jupyter>=1.0.0", "papermill>=1.2.1",],},
```

Comment:
Great idea to add test-related stuff as extras :+1: 

I would also move:

        "black>=19.10b0",
        "pytest>=5.3.5",
        "pytest-black>=0.3.8",

from `install_requires` to `extra_requires` and also format them in the same way as `install_requires`. One package per line.

Reply:
done.